### DISCUSSION TITLE
corfu--auto-complete-deferred warning

### DISCUSSION THREAD

**Discussion by danielpopescu** [minad/corfu#475] [Q&A] [answered]
I use corfu/cape for everything (global-corfu-mode turned on), but in org (org-babel mode) I use 

(use-package company-org-block
  :ensure t
  :custom
  (company-org-block-edit-style 'inline) ;; 'auto, 'prompt, or 'inline
  :hook ((org-mode . (lambda ()
                       (setq-local company-backends '(company-org-block))
                       (company-mode +1)))))

so I get org-block autocompletion....

This worked fine, but starting few versions ago, I get:

Error running timer ‘corfu--auto-complete-deferred’: (error "‘cape-company-to-capf’ should not be used with ‘company-mode’, use the Company backend directly instead") [2 times]

anyway way to get around it (silence this warning)? BTW, I realize that this may be a cape question...

TY

**Comment by minad** [comment, marked as answer]
You should make sure that you don't enable both Corfu and Company in the same buffer. You could also try to use `company-org-block` with `cape-company-to-capf` and Corfu or alternatively use a Capf equivalent for org-block if this exists.

**Comment by danielpopescu** [comment]
Ty for the quick reply. I did try:

(use-package company-org-block
  :ensure t
  :custom
  (company-org-block-edit-style 'inline) ;; 'auto, 'prompt, or 'inline
  :hook ((org-mode . (lambda ()
                       (setq-local company-backends '(company-org-block))
                       (corfu-mode -1)
                       (company-mode +1)
                       ))))


But it did not do anything useful....
